### DISCUSSION TITLE
Problem with Sandboxing Setting

### DISCUSSION THREAD

**Discussion by Mythix-99** [apohl79/audiogridder#1218] [Help]
So for the first time, I seem to have choked my system since using audiogridder   :-(
In the process of fiddling with settings, I disabled the sandboxing. After looking into it more, and realizing it will be worse for me, I tried to switch back to Plugin Isolation. Now, in the settings GUI, it only ever has "Total Workers" and never shows sandboxing anymore. Is this just a GUI bug? Or does switching back simply not work as it should? Have rebooted the system and audiogridder a hundred times. It's still shows as though it is not sandboxing anymore.

**Comment by apohl79** [comment]
Can you post a screen shot? Total workers would be a label in statistics window, rather then the server settings.

**Reply by Mythix-99** [reply]
Yeah, sorry I meant in the statistics window. I'm still trying to figure out what's causing the the random CPU spikes as well. Everything works fine until I open a plugin, and then all hell breaks loose, and sometimes it doesn't recover without shutting down Cubase and reopening it. Very annoying. What network speeds would you expect to be seeing between the systems? It feels like peaking at around 20mb/s is extremely slow. Wondering if I have a bottle neck there throttling my performance.

![image](https://github.com/apohl79/audiogridder/assets/97277167/245b2db4-c0ef-46a6-bebf-600982ce24d0)

![image](https://github.com/apohl79/audiogridder/assets/97277167/d5735084-53a5-45bb-bf5e-d6d976fae92d)
